### PR TITLE
Feature/operator extanded support

### PR DESCRIPTION
**Pull request by yuvdar**
Using these tool I experienced two issues. This PR fix them.

1. A segmentation fault in the C implementation of infer_shape. The solution was to run the infer shape on the model itself and not on the graph (see first commit)

2. Some layers in onnx had different field names than the one assumed in the code. The second and third commit handle those

### CODE REVIEW

File: onnx_coreml/converter.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def _set_deprocessing(is_grayscale,  # type: bool
 
 
 def _prepare_onnx_graph(graph, transformers):  # type: (Graph, Iterable[Transformer]) -> Graph
-    graph = infer_shapes_and_types(graph)
+    # graph = infer_shapes_and_types(graph)
```
Comment (aseemw) [inline onnx_coreml/converter.py]: If we are not using this function then I guess the file "_shapeinference.py" should be removed.
Can you do that as well in this PR?

**Comment by aseemw** [comment]
Looks good to me!

File: onnx_coreml/converter.py (hunk 1 above)
Comment (yuvdar) [reply onnx_coreml/converter.py]: removed.

**Comment by yuvdar** [comment]
The file removed. `from onnx import shape_inference` was added instead.

File: onnx_coreml/converter.py (hunk 1 above)
Comment (aseemw) [reply onnx_coreml/converter.py]: Cool!

**Comment by aseemw** [comment]
We can merge it once Travis passes.

**Pull request merged**